### ISSUE TITLE
A naive question

### ISSUE THREAD

**Issue by dadada101** [facebookresearch/MaskFormer#27]
hi, @bowenc0221 
   a naive question  in https://github.com/facebookresearch/MaskFormer/blob/2ae8543c892c5d57352fc10d55103a7254a03dad/mask_former/modeling/criterion.py#L101, why the src_logits should be transposed?
  Thanks.

**Comment by bowenc0221** [comment]
The second dimension should be "channel".

Please check the PyTorch document on `F.cross_entropy` usage: https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html#torch.nn.functional.cross_entropy

**Comment by WalBouss** [comment]
Hi,
Thank you for your great job and for sharing the code!

I wonder why are you using gradient clipping during training, with a very low max norm `CLIP_VALUE=0.01`
https://github.com/facebookresearch/MaskFormer/blob/2ae8543c892c5d57352fc10d55103a7254a03dad/configs/ade20k-150/Base-ADE20K-150.yaml#L29-L33

Have you done some experiments and find out that such a low value improves the training?]
Thanks!

**Comment by bowenc0221** [comment]
We follow the setting in the original DETR implementation, but based on my experience gradient clipping does not affect MaskFormer performance.
